src/epub/create_epub.py

```diff
@@ -0,0 +1,165 @@
+import os
+import zipfile
+
+from lxml import etree
+import shutil
+from zipfile import ZipFile
+from datetime import datetime
+
+import src.html.create_songs_html as cash
+
+
+class SongMeta:
+    def __init__(self, title='', alias='', plik=''):
+        self.title = title if title else ''
+        self.alias = alias if alias else ''
+        self.plik = plik
+
+    # def __repr__(self):
+    #     return "(%s, alias: %s, plik: %s)" % (self.title, self.alias, self.plik)
+
+    @staticmethod
+    def parseDOM(root, plik):
+        def elementTextOrNone(elem):
+            return elem.text if elem is not None else None
+        return SongMeta(
+            title=root.get('title'),
+            alias=elementTextOrNone(root.find('{*}alias')),
+            plik=plik
+        )
+
+
+def add_song(path, lista):
+    tree = etree.parse("../../songs/" + path + ".xml")
+    plik = path + ".xml"
+    song = SongMeta.parseDOM(tree.getroot(), plik)
+    lista.append(song)
+
+
+def list_of_song(path_out):
+    songs_list = os.listdir(path_out)
+    list_od_meta = []
+    for song in songs_list:
+        if song[-5:] == '.html':
+            path = song[0:-5]
+            add_song(path, list_od_meta)
+    list_od_meta.sort(key=lambda x: x.title)
+    return list_od_meta
+
+
+def create_content_opf(list_of_songs_meta, target_dir):
+    tmp_path = 'templates/content.opf'
+    out_path = os.path.join(target_dir, "epub", "OEBPS", "content.opf")
+    tree = etree.parse(tmp_path)
+    root = tree.getroot()
+    metadata = root.getchildren()[0]
+    title = metadata.getchildren()[0]
+    title.text += str(datetime.now().strftime("%d/%m/%Y %H:%M"))
+    manifest = root.getchildren()[1]
+    spine = root.getchildren()[2]
+    for i in range(len(list_of_songs_meta)):
+        x = etree.SubElement(manifest, "item")
+        x.attrib['id'] = 'p' + str(i + 1)
+        x.attrib['href'] = list_of_songs_meta[i].plik[:-4] + '.html'
```

Comment:
Case for: 
https://docs.python.org/3/library/os.path.html#os.path.splitext

Reply:
done